resources/filter_lines.rb

```diff
@@ -45,7 +45,7 @@
   # filter ::= <code> | { <code> => <args> }
   # args ::= <String> | <Array>
   # code ::= <Symbol> | <Method> | <Proc>
-  # Symbol ::= :after | :before | :between | :comment | :missing | :replace | :stanza | :substitute
+  # Symbol ::= :after | :before | :between | :comment | :delete_between | :missing | :replace | :stanza | :substitute
```

Comment:
Test failing on this file:
```
    TypeError
    ---------
    no implicit conversion of nil into String
    
    Cookbook Trace:
    ---------------
    /opt/kitchen/cache/cookbooks/line/resources/filter_lines.rb:34:in `exist?'
    /opt/kitchen/cache/cookbooks/line/resources/filter_lines.rb:34:in `block in class_from_file'
```

Reply:
Very much still work in progress.  The stanza filter has to be merged before this one.  Thanks  for looking at it.

Reply:
Should be fixed now.